tests/system/Router/RouteCollectionTest.php

```diff
@@ -819,6 +819,16 @@ public function testCLI()
         $this->assertSame($expected, $routes->getRoutes('cli'));
     }
 
+    public function testView()
+    {
+        $routes = $this->getCollector();
+
+        $routes->view('here', 'hello');
+
+        $route = $routes->getRoutes('get')['here'];
+        $this->assertIsCallable($route);
+    }
+
```

Comment:
Can we get something like `testViewRespondsToGetOnly()` that shows a POST resulting in 404?

Reply:
I've updated that test to verify it's only available in the GET routes, not in any of the others (like POST, HEAD, etc). Assuming all of the other tests in our system are correct, this should prove the same thing. :)

Reply:
Even better!